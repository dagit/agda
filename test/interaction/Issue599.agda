module Issue599 where

data Bool : Set where
  true false : Bool

-- standard lambda here
foo : Bool → Bool
foo = ?

-- pattern matching lambda here
bar : Bool → Bool
bar = ?

