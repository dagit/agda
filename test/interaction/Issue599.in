top_command (cmd_load_highlighting_info currentFile)
top_command (cmd_load currentFile [])
--top_command (cmd_load_highlighting_info currentFile)
goal_command 0 (cmd_refine_or_intro False) ""
goal_command 1 (cmd_refine_or_intro True) ""
