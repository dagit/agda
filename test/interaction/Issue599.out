Prelude> Agda2> (agda2-status-action "")
Agda2> (agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" "Checking Issue599 (Issue599.agda).\n" t)
(agda2-info-action "*Type-checking*" "Finished Issue599.\n" t)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?0 : Bool → Bool\n?1 : Bool → Bool\n" nil)
((last . 1) . (agda2-goals-action '(0 1)))
Agda2> Agda2> (agda2-give-action 0 "λ x → ?")
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?1 : Bool → Bool\n?2 : Bool\n" nil)
((last . 1) . (agda2-goals-action '(2 1)))
Agda2> (agda2-give-action 1 "λ {x → ?}")
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?2 : Bool\n?3 : Bool\n" nil)
((last . 1) . (agda2-goals-action '(2 3)))
Agda2> 